README.md

```diff
@@ -1 +1,65 @@
-# Leaderboard
\ No newline at end of file
+# Project Name
+
+Leaderboard
+
+# PR
+
+1-Add items to the Leaderboard
+2-send Add items to an API
+![screenshot](./Capture.PNG)
+
+## Built With
+
+- Html
+- css
+- JavaScript
+
+## live demo :
+-not yet
+
+## Getting Started
+
+-open the index.html file in chrome and see the Leaderboard
+
+### Setup
+
+-click the code button in the repo and then download the files as a zip file
+
+#### To get a local copy up and running follow these simple example steps.
+
+- Click the green `Code` button on the repo and copy link
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/MeqdamAlqudah/Leaderboard`
+- On the terminal, navigate into the directory like so: `cd Leaderboard`
+
```

Comment:
- [ ] **[OPTIONAL]** Users cannot run your app without first running `npm i` and `npm start`. It would be nice to add these specific instructions